iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/execution/relational/ColumnTransformerBuilder.java

```diff
@@ -996,6 +997,16 @@ private ColumnTransformer getFunctionColumnTransformer(
           source,
           ((LongLiteral) children.get(3)).getParsedValue(),
           context.sessionInfo.getZoneId());
+    } else if (TableBuiltinScalarFunction.FORMAT.getFunctionName().equalsIgnoreCase(functionName)) {
+      List<ColumnTransformer> columnTransformers = new ArrayList<>();
+      for (int i = 1; i < children.size(); i++) {
+        columnTransformers.add(this.process(children.get(i), context));
+      }
+      return new FormatColumnTransformer(
+          STRING,
+          ((StringLiteral) children.get(0)).getValue(),
```

Comment:
must first argument be StringLiteral? It can also be any expression that is String/Text type